test/test_all.py

```diff
@@ -154,6 +154,8 @@ def __init__(self, arch, container, address, *args, **kwargs):
                                                         (14, 1), (15, 1)))
                            for fname in fnames])
 
+testset += RegressionTest(["mem.py"], base_dir="analysis")
```

Comment:
Right. Before I add it, do you think `example/jitter/memstruct.py` is a correct path for it? I can't figure out where it would fit the best.